src/devices/src/virtio/vsock/mod.rs

```diff
@@ -120,6 +120,8 @@ pub enum VsockError {
     /// Invalid virtio configuration.
     VirtioState(VirtioStateError),
     VsockUdsBackend(VsockUnixBackendError),
+    /// Empty queue
+    EmptyQueue,
```

Comment:
nit: ideally, we want to have `enum` variants in alphabetical order, to better find and navigate.